app/core/security.py

```diff
@@ -38,10 +41,10 @@ def create_token_with_id(self, userId):
         """Creates a JWT token with the `id` of the `user`."""
         token = jwt.encode(
             payload={
-                "subject": userId,
-                "expire_at": (datetime.utcnow() + timedelta(minutes=30)).ctime(),
+                "sub": userId,
+                "exp": (datetime.utcnow() + timedelta(minutes=2)).timestamp(),
             },
-            key=config["SECRET_KEY"],  # type: ignore
+            key=settings.SECRET_KEY,  # type: ignore
```

Comment:
settings.JWT_SECRET_KEY may be better used instead of settings.SECRET_KEY

Reply:
noted ✅